src/Common/src/Interop/Windows/NtDll/Interop.NtCreateFile.cs

```diff
@@ -33,7 +36,23 @@ internal partial class NtDll
             System.IO.FileShare shareAccess = System.IO.FileShare.ReadWrite | System.IO.FileShare.Delete,
             System.IO.FileAttributes fileAttributes = 0,
             CreateOptions createOptions = CreateOptions.FILE_SYNCHRONOUS_IO_NONALERT,
-            ObjectAttributes objectAttributes = ObjectAttributes.OBJ_CASE_INSENSITIVE)
+            ObjectAttributes objectAttributes = ObjectAttributes.OBJ_CASE_INSENSITIVE
+            )
+        {
+                return ( CreateFile(eaBuffer: new Microsoft.Win32.SafeHandles.SafeFileHandle(IntPtr.Zero, true), eaLength: 0, path, rootDirectory,createDisposition, desiredAccess, shareAccess, fileAttributes, createOptions, objectAttributes) );
+        }
+        internal unsafe static (int status, IntPtr handle) CreateFile(
```

Comment:
Had to create a separate CreateFile Method as SqlFileStream requires the Extended Attributes buffer to be supplied and currently there is currently no way to do so other than NTCreateFile #29379